src/config/ssd.md

```diff
@@ -0,0 +1,150 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If DISC-GRAN (discard granularity) and DISC-MAX (discard max bytes) columns are
+non-zero, that means the block device has TRIM support. If your solid state
+drive partition does not show TRIM support, please verify that you chose a file
+system with TRIM support (ext4, btrfs, etc.).
```

Comment:
For completion's sake, and because it was created specifically for flash memory, it would be cool to list F2FS filesystem too.

Reply:
Just added a note before running fstrim about having kernel 4.19 or greater as that is the only requirement for using f2fs trim.